test/hotspot/jtreg/runtime/Monitor/MonitorUsedDeflationThresholdTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2021, 2023, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2021, 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Reviewed this file with whitespace changes disabled. Hopefully I
didn't miss anything that way...

Reply:
My editor strips trailing blanks and my commit command updates copyrights, so you should have seen any other weird formatting in this diff.